requirements.txt

```diff
@@ -0,0 +1,102 @@
+amqp==2.4.1
```

Comment:
why is this file necessary?

Reply:
I pushed when I was using pip instead of pipenv at home, but now it's deleted from the repository.